### DISCUSSION TITLE
where is torch.jit.module forward method?

### DISCUSSION THREAD

**Discussion by dbriard** [xamarin/TorchSharp#1108] [Q&A] [answered]
Hi, I am new to TorchSharp and I am trying to port my C++ code to C#.

I found how to load my model the same way as in C++, but the forward method is missing?
![image](https://github.com/dotnet/TorchSharp/assets/3595443/5f161290-d231-4232-9e58-1e79145d24d6)

The documentation (here https://github.com/dotnet/TorchSharp/wiki/TorchScript) say that Script modules have a main forward() method.

Can someone help? Thanks

**Comment by NiklasGustafsson** [comment, marked as answer]
It's `call()` in TorchSharp -- it corresponds to treating a module as a function object in Python. `forward()` is missing on ScriptModule, apparently.

**Reply by dbriard** [reply]
Thanks a lot @NiklasGustafsson  I would never have guessed, and it is a bit weird to me that TorchSharp do not use the original name of such important function. May be the doc I mention in my post above should be updated.

![image](https://github.com/dotnet/TorchSharp/assets/3595443/8d55c7a5-28ec-4b06-a036-d855391ec4a3)

**Reply by NiklasGustafsson** [reply]
It was an oversight and will be fixed. In the meantime, please use 'call'

**Reply by dbriard** [reply]
Works perfectly! TorchSharp has replaced my old C++/CLI project :)

**Comment by NiklasGustafsson** [comment]
See: https://github.com/dotnet/TorchSharp/pull/1110
